src/executables/util_cs.c

```diff
@@ -3275,51 +3280,60 @@ applyinfo (UTIL_FUNCTION_ARG * arg)
 	    {
 	      goto check_applied_info_end;
 	    }
-	  error = db_restart (arg->command_name, TRUE, local_database_name);
-	  if (error != NO_ERROR)
+	  if ((error = db_restart (arg->command_name, TRUE, local_database_name)) != NO_ERROR)
 	    {
 	      goto check_applied_info_end;
 	    }
 
 	  if (HA_DISABLED ())
 	    {
+	      check_applied_info = false;
+	      printf ("\nERROR : ");
 	      PRINT_AND_LOG_ERR_MSG (msgcat_message (MSGCAT_CATALOG_UTILS, MSGCAT_UTIL_SET_APPLYINFO,
 						     APPLYINFO_MSG_NOT_HA_MODE));
 	      goto check_applied_info_end;
 	    }
-
-	  error = la_log_page_check (local_database_name, log_path, pageid, check_applied_info, check_copied_info,
-				     check_replica_info, verbose, &copied_eof_lsa, &copied_append_lsa,
-				     &applied_final_lsa);
-	  (void) db_shutdown ();
-	}
-      else if (check_copied_info)
-	{
-	  memset (local_database_name, 0x00, CUB_MAXHOSTNAMELEN);
-	  strcpy (local_database_name, database_name);
-	  strcat (local_database_name, "@localhost");
-
-	  error = la_log_page_check (local_database_name, log_path, pageid, check_applied_info, check_copied_info,
-				     check_replica_info, verbose, &copied_eof_lsa, &copied_append_lsa,
-				     &applied_final_lsa);
+	  error =
+	    la_get_applyinfo_applied_log_info (database_name, log_path, check_replica_info, verbose,
```

Comment:
`applyinfo` utility doesn't do anything with the applied_log_info returned from the `la_get_applyinfo_..`.
how about modifying the function name to `la_show_..` or `la_print_..` ?

Reply:
applied_final_lsa works for `Delay in Applying Copied Log` from  ha_apply_info

Reply:
Then, the comment for `applied_final_lsa` is wrong. it is just described as input parameter. ( 7052L in log_applier.c)

Reply:
The direction is not enough description. the (in) must be edited with (in/out).

I will check any others. Thank you.